.github/workflows/win64_llvm_clang_build_and_cache.yml

```diff
@@ -0,0 +1,54 @@
+name: Build CMake Libraries
+
+on:
+  workflow_dispatch:
+
+env:
+    LLVM_VER: 16.0.4
+    BRANCH: release/16.x

```

Comment:
we should use version 16.0.6 and the tag `llvmorg-16.0.6` as that is the newest patch release